data/assets/scene/solarsystem/planets/uranus/major_moons.asset

```diff
@@ -19,7 +19,7 @@ local Miranda = {
     SegmentsPerPatch = 64,
     Layers = {}
   },
-  Tag = { "moon_solarSystem", "moon_giants", "moon_uranus", "moon_major" },
+  Tag = { "moon_solarSystem", "moon_giants", "moon_uranus", "moon_major", "moon_major_uranus"},
```

Comment:
Lost a space at the end

Reply:
Fixed in bf3fd0a20898eb76105d1878c13baf69aec5b21d